opcua.cpp

```diff
@@ -1697,7 +1707,16 @@ OPCUA::Node::Node(uint32_t conn, const string &nodeId) : m_nodeID(nodeId)
 	readValue[2].indexRange = NULL;
 
 	int res;
-	if ((res = SOPC_ClientHelper_Read(conn, readValue, 3, values)) == 0)
+	int RETRY_COUNT=5;
```

Comment:
I would prefer a lower case variable name here. Uppercase names are commonly used for macro names (i.e.#define constants) or enum values.